src/node_contextify.cc

```diff
@@ -65,6 +66,12 @@ using v8::WeakCallbackInfo;
 
 namespace {
 
+v8::Local<v8::Name> UIntToName(v8::Isolate* isolate, uint32_t index) {
+  const char* x = std::to_string(index).c_str();
```

Comment:
If this function gets called a lot (which it looks like it is), it's arguably better to use `snprintf()` instead to avoid the extra allocation, or even to do manual stringification.  C++17 adds `std::to_chars()` but we can't use that yet, unfortunately.

It might even be worthwhile to implement this as `Uint32::New(...)->ToString(...)` because that would reuse V8's number-to-string cache.

Reply:
Yes, if there's any larger array, this is called for every index. Using `Uint32::New()->ToString()`.